dspace-server-webapp/src/main/java/org/dspace/app/rest/security/EPersonRestAuthenticationProvider.java

```diff
@@ -47,6 +48,8 @@
 
     private static final Logger log = LoggerFactory.getLogger(EPersonRestAuthenticationProvider.class);
 
+    public static final String ACCOUNT_ADMIN_GRANT = "ACCOUNT_ADMIN";
```

Comment:
Can we rename this constant ? ACCOUNT_ADMIN_GRANT doesn't really point to the fact that the user could be a community / collection admin.

Reply:
I agree, `MANAGE_ACCESS_GROUPS` looks indeed a better name